build_tools/customizations.rb

```diff
@@ -120,27 +120,32 @@ def dynamodb_example_deep_transform(subsegment, keys)
 
     end
 
-    api('RDS') do |api|
-      %w(
-        CopyDBSnapshotMessage
-        CreateDBInstanceReadReplicaMessage
+    # Cross Region Copying
+    %w[RDS Neptune DocDB].each do |service|
+      operations = %w[
         CopyDBClusterSnapshotMessage
         CreateDBClusterMessage
-      ).each do |shape_name|
-        api['shapes'][shape_name]['members']['DestinationRegion'] = {"shape" => "String", "documented" => false }
-        api['shapes'][shape_name]['members']['SourceRegion'] = {"shape" => "String"}
+      ]
+      if service == 'RDS'
+        operations += %w[
```

Comment:
Rather than having RDS add to this list, It might be better to have a map from service => operations that is static.

Reply:
these three services share an API (same endpoint), so if we were to do this, operations would be duplicated. i think it makes sense to just make a special case for RDS and refactor away later if needed